test/Libraries/SystemTestServices/SystemTestBase.cs

```diff
@@ -67,7 +67,9 @@ public virtual void Setup()
             if (assemblyResolver == null)
             {
                 assemblyResolver = new AssemblyResolver();
-                assemblyResolver.Setup(testConfig.DynamoCorePath);
+                assemblyResolver.Setup(testConfig.DynamoCorePath, new [] {
```

Comment:
same as in another PR - would be good to check this does not cause issues on master-15